src/components/Pagination/Pagination.scss

```diff
@@ -8,6 +8,7 @@ $difference-between-touch-area-and-backdrop: control-height() -
 // This is the adjustment needed to get the icon to appear flush with other
 // content around it.
 $plain-horizontal-adjustment: spacing(tight);
+$labelMinWidth: rem(80px);
```

Comment:
Why did you choose `80px` for this? Is this based on the expected min width of text that follows the format `number of number`?

Reply:
I agree with @LauraAubin and would prefer this to be the width of the content with padding.

Reply:
That's a much better idea I'll make that change!